selfdrive/locationd/laikad.py

```diff
@@ -391,13 +391,16 @@ def process_msg(laikad, gnss_msg, mono_time, block=False):
 
 
 def clear_tmp_cache():
+  if "CI" in os.environ or "REPLAY" in os.environ:
+    return
```

Comment:
why does CI rely on this?

Reply:
replay uses CI env flag, and runs it in parallel, thats doesnt work

Reply:
hmm, can you add a comment to remove this once we're not writing this cache to disk? CI exceptions should be used extremely sparingly and only when absolutely necessary